src/Apps/Settings/Routes/Shipping/Components/SettingsShippingAddressForm.tsx

```diff
@@ -40,16 +38,20 @@ const INITIAL_VALUES = {
 
 type Address = typeof INITIAL_ADDRESS
 
-const validationSchema = Yup.object().shape({
+const VALIDATION_SCHEMA = Yup.object().shape({
   attributes: Yup.object().shape({
     name: Yup.string().required("Name is required"),
     country: Yup.string().required("Country is required"),
     addressLine1: Yup.string().required("Address is required"),
     city: Yup.string().required("City is required"),
     region: Yup.string().required("Region is required"),
     postalCode: Yup.string().required("Postal Code is required"),
-    phoneNumber: Yup.string().required("Please enter a valid phone number"),
```

Comment:
This previously was a message for the requirement; not the validation. Performing custom validation requires an additional field that we'll assert against and raise an error against.